stubs/zxcvbn/zxcvbn/time_estimates.pyi

```diff
@@ -0,0 +1,24 @@
+from decimal import Decimal
+from typing_extensions import Literal, TypedDict
+
+class _TimeEstimate(TypedDict):
+    crack_times_seconds: _CrackTimeSeconds
+    crack_times_display: _CrackTimesDisplay
+    score: Literal[0, 1, 2, 3, 4]
+
+class _CrackTimeSeconds(TypedDict):
+    online_throttling_100_per_hour: Decimal
+    online_no_throttling_10_per_second: Decimal
+    offline_slow_hashing_1e4_per_second: Decimal
+    offline_fast_hashing_1e10_per_second: Decimal
+
+class _CrackTimesDisplay(TypedDict):
+    online_throttling_100_per_hour: str
+    online_no_throttling_10_per_second: str
+    offline_slow_hashing_1e4_per_second: str
+    offline_fast_hashing_1e10_per_second: str
+
+def estimate_attack_times(guesses: Decimal) -> _TimeEstimate: ...
```

Comment:
Are you sure this has to be a `Decimal`? It looks like the `Decimal` constructor is called on `guesses` inside this function, which would be a bit odd if it has to already be a `Decimal`: https://github.com/dwolfhub/zxcvbn-python/blob/f596bef499c02bb6f0c8aef33b260364add2dc2b/zxcvbn/time_estimates.py#L5-L8

Reply:
Whoops, I was under the (mistaken) assumption that `float` is a subtype of `Decimal`.

Reply:
I'm skipping the other possible values except `float` here, since I don't think they are relevant.